### PR TITLE
fixing warnings

### PR DESCRIPTION
**Pull request by mikeallbutt**
WARNING: Title underline too short.
WARNING: Document may not end with a transition.
WARNING: Inline literal start-string without end-string.
WARNING: Title level inconsistent

### CODE REVIEW

**Comment by corsacca** [comment]
Can you summarize everything you've added? 
I see new pages like archive

**Comment by mikeallbutt** [comment]
I did 3 commits that fixed errors like having unlinked pages.
And added a page to start documentation about the app.
